.github/workflows/build_and_test.yaml

```diff
@@ -32,65 +32,23 @@ jobs:
           name: rock
           path: ${{ steps.rockcraft.outputs.rock }}
 
-  CephAdmScriptTest:
+  flake8-lint:
     runs-on: ubuntu-latest
-    needs: build-rock
-    strategy:
-      matrix:
-        python-version: ["3.10"]
-
+    name: Lint
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v3
-      - name: clean unrequired files.
-        run: |
-          sudo rm -rf /usr/share/dotnet
-          sudo rm -rf /opt/ghc
-          sudo rm -rf "/usr/local/share/boost"
-          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
```

Comment:
istr that we were running out of space when not cleaning up, hope that's not an issue anymore?

Reply:
With LXD VMs providing the block device for ceph osds we do not get out of storage. The original issue was also around the single available block device (we need 3) which became trickier because the disk layout kept changing.

Reply:
These files are still irrelevant but I decided not to perform a cleanup unless necessary.